lib/modules/manager/flux/extract.spec.ts

```diff
@@ -57,7 +57,7 @@ describe('modules/manager/flux/extract', () => {
     it('extracts version and components from system manifests', () => {
       const result = extractPackageFile(
         Fixtures.get('flux-system/gotk-components.yaml'),
-        'clusters/my-cluster/flux-system/gotk-components.yaml'
+        'clusters/my-cluster/gotk-components.yaml'
```

Comment:
i would prefer to add a new test instead of changing existing

Reply:
I've refactored this to do a matrix of file paths to cover both the new and old behaviour as copy-pasting the expected object doesn't feel very DRY. Happy to do that if it's more the expected convention though 👍